services/galley/src/Galley/Cassandra/Client.hs

```diff
@@ -60,13 +62,24 @@ eraseClients user = retry x5 (write Cql.rmClients (params LocalQuorum (Identity
 interpretClientStoreToCassandra ::
   ( Member (Embed IO) r,
     Member (Input ClientState) r,
-    Member (Input Env) r
+    Member (Input Env) r,
+    Member TinyLog r
   ) =>
   Sem (ClientStore ': r) a ->
   Sem r a
 interpretClientStoreToCassandra = interpret $ \case
-  GetClients uids -> embedClient $ lookupClients uids
-  CreateClient uid cid -> embedClient $ updateClient True uid cid
-  DeleteClient uid cid -> embedClient $ updateClient False uid cid
-  DeleteClients uid -> embedClient $ eraseClients uid
-  UseIntraClientListing -> embedApp . view $ options . settings . intraListing
+  GetClients uids -> do
+    debug (msg (val "ClientStore.GetClients"))
```

Comment:
I'm wondering if we shouldn't reduce the duplication and transform this

```
debug (msg (val "ClientStore.GetClients"))
```

into e.g.

```
logEffect "ClientStore.GetClients"

logEffect :: Member TinyLog r -> String -> Sem r a
logEffect = debug . msg . val
```
:thinking: 

However, this is likely a matter of taste. (I was able to read your approach, too :wink: )

Reply:
Can do.

Reply:
Done.